api/lib/domain/read-models/AllowedCertificationCenterAccess.js

```diff
@@ -0,0 +1,63 @@
+const { features } = require('../../config');
+
+class AllowedCertificationCenterAccess {
+
+  constructor({
+    certificationCenterId,
+    certificationCenterName,
+    certificationCenterExternalId,
+    certificationCenterType,
+    isRelatedToManagingStudentsOrganization,
+    relatedOrganizationTags,
+  }) {
+    this.certificationCenterId = certificationCenterId;
+    this.certificationCenterName = certificationCenterName;
+    this.certificationCenterExternalId = certificationCenterExternalId;
+    this.certificationCenterType = certificationCenterType;
+    this.isRelatedToManagingStudentsOrganization = isRelatedToManagingStudentsOrganization;
+    this.relatedOrganizationTags = relatedOrganizationTags;
+  }
+
+  isAccessBlockedCollege() {
+    return this.isCollege()
+      && !this.isMulticentre()
+      && !this.isInWhitelist()
+      && new Date() < new Date('2022-02-01');
+  }
+
+  isAccessBlockedLycee() {
+    return this.isLycee()
+      && !this.isMulticentre()
+      && !this.isInWhitelist()
+      && new Date() < new Date('2021-11-01');
+  }
+
+  hasTag(tagName) {
+    return this.relatedOrganizationTags.includes(tagName);
+  }
+
+  isCollege() {
+    return this.certificationCenterType === 'SCO'
+      && this.isRelatedToManagingStudentsOrganization
+      && this.hasTag('COLLEGE');
+  }
```

Comment:
On pourrait reprendre les getters comme pour _certif/app/models/allowed-certification-center-access.js_
```suggestion

  get isScoManagingStudents() {
    return this.certificationCenterType === 'SCO' && this.isRelatedToManagingStudentsOrganization;
  }
  
  isCollege() {
    return this.isScoManagingStudents() && this.hasTag('COLLEGE');
  }
  
    isLycee() {
    return this.isScoManagingStudents() && (this.hasTag('LYCEE') || this.hasTag('LYCEE PRO'));
  }

  isMulticentre() {
    return this.isScoManagingStudents() && this.isLycee() && this.isCollege();
  }
```

Reply:
factoriser l'ensemble isScoManagingStudents

Reply:
Done